corehq/apps/reports/standard/monitoring.py

```diff
@@ -53,17 +53,16 @@
 from corehq.util import flatten_list
 from corehq.util.timezones.conversions import ServerTime, PhoneTime
 from corehq.util.view_utils import absolute_reverse
-from django.conf import settings
 from dimagi.utils.couch.safe_index import safe_index
 from dimagi.utils.dates import DateSpan, today_or_tomorrow
 from dimagi.utils.decorators.memoized import memoized
 from dimagi.utils.parsing import json_format_date, string_to_utc_datetime
-from django.utils.translation import ugettext as _, ugettext_lazy
-from django.utils.translation import ugettext_noop
+
 import six
 from functools import reduce
 from six.moves import range
 from six.moves import map
+from six.moves.urllib.parse import urlencode
```

Comment:
Init  think pep8 is formatted as

```
std library
third party libs
your app
```

So I think functools should be at the top and six next to django right?

Reply:
I was thinking of this as part of a 2-to-3 compatibility block, although I guess it's technically a stdlib import.  IDK, no strong feelings on it.